packages/help-center/src/data/use-submit-support-ticket.ts

```diff
@@ -10,6 +10,8 @@ type Ticket = {
 	is_chat_overflow: boolean;
 	source: string;
 	blog_url: string;
+	ai_id?: string;
```

Comment:
What do you think about using `ai_answer_id` to make it clearer what this is? And it's close to the source `ai_answer` field. Otherwise, "AI ID" might be vague (is it the ID of the "AI", meaning the ID of the bot, model, etc.?).